test/integration/default/inspec/default_spec.rb

```diff
@@ -0,0 +1,7 @@
+describe http('http://localhost:4000/directory', enable_remote_worker: true) do
+  its('body') { should match(%r{"new-cert": "http://(localhost|127.0.0.1):4000/acme/new-cert"}) }
+end
+
+describe http('https://foo.org', enable_remote_worker: true, ssl_verify: false) do
```

Comment:
This won't work because it's trying to access the real server at foo.org instead of locally. You need to change this to the following:
``` ruby
describe http(
  'https://localhost',
  headers: { 'Host' => 'foo.org' },
  enable_remote_worker: true,
  ssl_verify: false
) do
```

This sets the ``Host`` header which sets the virtual host for the web server. This makes the server think that's the site we're wanting to access.

Reply:
So I was wrong and this needs to retain the original command check using curl. It doesn't look like we can use the ``ssl`` inspec resource in a manner we need.